modules/utxo-lib/test/transaction_util.ts

```diff
@@ -66,12 +67,25 @@ export function mockTransactionId(v = 0xff): string {
   return Buffer.alloc(32).fill(v).toString('hex');
 }
 
-export function getPrevOutput(
+export function toTNumber<TNumber extends number | bigint>(v: any | undefined, t: 'number' | 'bigint'): TNumber {
```

Comment:
```suggestion
export function toTNumber<TNumber extends number | bigint>(v: number | bigint | undefined, t: 'number' | 'bigint'): TNumber | undefined {
```

Reply:
I had copied the similar function in `bitcoinjs-lib`, but it turns out that v shouldn't ever be `undefined`.